app/waf/src/controllers/member.controller.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  post,
+  param,
+  get,
+  getFilterSchemaFor,
+  getWhereSchemaFor,
+  patch,
+  put,
+  del,
+  requestBody,
+} from '@loopback/rest';
+import {Member} from '../models';
+import {MemberRepository} from '../repositories';
+import uuid = require('uuid');
+
+const prefix = '/adcaas/v1';
+
+export class MemberController {
+  constructor(
+    @repository(MemberRepository)
+    public memberRepository: MemberRepository,
+  ) {}
+
+  @post(prefix + '/members', {
```

Comment:
The member resource of lbass v2 API looks like "/v2/lbaas/pools/{pool_id}/members". Suggest to be consistent with it.

Reply:
I just read the code, it probably needs to be added into the pool controller.